src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/customize_panel/customize_panel_editor.tsx

```diff
@@ -170,9 +176,16 @@ export const CustomizePanelEditor = (props: CustomizePanelProps) => {
               size="xs"
               data-test-subj="resetCustomEmbeddablePanelDescriptionButton"
               onClick={() => {
-                setPanelDescription(embeddable.getOutput().defaultDescription);
+                setPanelDescription(
+                  embeddable.getInput().description ?? embeddable.getOutput().defaultDescription
+                );
               }}
-              disabled={hideTitle || !editMode}
+              disabled={
+                hideTitle || !editMode || embeddable.getInput().description != undefined
+                  ? embeddable.getInput().description === panelDescription
+                  : embeddable.getOutput().defaultDescription === panelDescription ||
+                    panelDescription === ''
+              }
```

Comment:
This should only be disabled if the panel description is equal to the default description:

```suggestion
             disabled={
                hideTitle ||
                !editMode ||
                embeddable.getOutput().defaultDescription === panelDescription
              }
```